src/go/k8s/pkg/resources/certmanager/pki.go

```diff
@@ -68,11 +68,8 @@ func (r *PkiReconciler) NodeCert() types.NamespacedName {
 
 // OperatorClientCert returns the namespaced name for the client certificate
 // used by the Kubernetes operator
-func (r *PkiReconciler) OperatorClientCert() *types.NamespacedName {
-	if !r.pandaCluster.Spec.Configuration.TLS.RequireClientAuth {
```

Comment:
Should we remove if statement?

Reply:
okay, I checked this change, it makes sense actually. Here we return the name at all times but in the statefulset we use it to mount only if client auth is enabled. I actually like this approach better so I would keep it like this

https://github.com/vectorizedio/redpanda/pull/875/files#diff-11cd0daf9c52813e733b2526ffd440b85ff6e9a8f6a11d8454c02cbd129bcb60R490